src/ng1/services/colorService/colorService.service.js

```diff
@@ -2,46 +2,61 @@ export default function $colorService() {
 
```

Comment:
Please reformat this file to consistently use spaces for indentation rather than tabs.